app/templates/components/forms/user-payment-info-form.hbs

```diff
@@ -0,0 +1,41 @@
+<form class="ui form" {{action 'submit' on='submit'}}> 
+  <h3 class="ui header">
+    {{t 'Payment Information'}}
+  </h3>
+  <div class="field">
+    <label class="required" for="contactName">{{t 'Contact Name'}}</label>
+    {{input type='text' id='contactName' value=data.billingContactName}}
```

Comment:
Do not mutate `authManager.currentUser` directly since it is used in a lot of places. Store all these data in an intermediate object then mutate  `authManager.currentUser` at the time of save.

Reply:
@niranjan94 i am using `mut` helper for this, will it be fine?

Reply:
No. Even when you use `mut` the `authManager.currentUser` is changing as the user types. it should change _only_ when save is pressed.